.vscode/settings.json

```diff
@@ -11,4 +11,4 @@
   "files.watcherExclude": {
     "**/target": true
   }
-}
+}
```

Comment:
The formatter is not happy about the missing ending newlines. You can run `yarn format` to fix it.

Reply:
Yeah, I did but I have a git hook that undid the work. 😄